lib/new_relic/agent/transaction.rb

```diff
@@ -719,9 +743,9 @@ def record_exceptions
       # Do not call this.  Invoke the class method instead.
       def notice_error(error, options = {}) # :nodoc:
         # Only the last error is kept
-        if @current_segment
-          @current_segment.notice_error error, expected: options[:expected]
-          options[:span_id] = @current_segment.guid
+        if current_segment
+          current_segment.notice_error error, expected: options[:expected]
```

Comment:
Would you be open to adding parens here?

Reply:
for sure, I didn't even notice the lack of them since i was eagle eye searching for `@current_segment` part of things lol.